.github/CODEOWNERS

```diff
@@ -1,55 +1,102 @@
 # GitHub CODEOWNERS definition
 # See: https://help.github.com/articles/about-codeowners/
 
-# * @elastic/beats
+# The beats repository is owned by the @elastic/elastic-agent-data-plane Many teams contribute to it. The goal
+# is to cover all directories in the CODEOWNERS file. The list is sorted alphabetically by directory and sub directories.
 
-# libbeat
-/libbeat/ @elastic/elastic-agent-data-plane
-# /auditbeat/ @elastic/beats
-# /packetbeat/ @elastic/beats
-# /filebeat/ @elastic/beats
-# /metricbeat/ @elastic/beats
-# /journalbeat/ @elastic/beats
-# /winlogbeat/ @elastic/beats
-
-# Auditbeat
-/auditbeat/module/ @elastic/security-external-integrations
+* @elastic/elastic-agent-data-plane
+
+
+### Private directories
+# Any changes to the CI system or .github must be reviewed by the data plane team.
+/.ci/ @elastic/elastic-agent-data-plane
+/.github/ @elastic/elastic-agent-data-plane
+
+### Auditbeat
+/auditbeat/ @elastic/security-external-integrations
 /x-pack/auditbeat/ @elastic/security-external-integrations
 
-# Packetbeat
-/packetbeat/protos/ @elastic/security-external-integrations
-/x-pack/packetbeat/ @elastic/security-external-integrations
+### Deploy
+# All changes to the different deployments must be reviewed by data plane team.
+# TODO: This where k8s falls
+/deploy/ @elastic/elastic-agent-data-plane
+
+# Beats Build Specific
+/dev-tools/ @elastic/elastic-agent-data-plane
 
-# Filebeat
-# /filebeat/module/ @elastic/integrations
+# Most docs change are inside each beat / module. For global docs, data plane team should review.
+/docs/ @elastic/elastic-agent-data-plane
+
+### Filebeat
+# Filebeat is owned by the data plan team
+# TODO: CLEANUP, go into more details
+/filebeat @elastic/elastic-agent-data-plane
+/x-pack/filebeat @elastic/elastic-agent-data-plane
+
+# Modules are by defaul owned by the integrations team
+/filebeat/module/ @elastic/integrations
```

Comment:
@masci Current most modules or owned by integrations or security team is my understanding? Does it make sense that the default ownership is with integrations?

Reply:
Happy to add a list with the detailed owners. @andrewkroh Are you ok with being the default owner?

Reply:
I think we should do the effort now of defining the module owners. It will reduce noise on the teams and prevent any ambiguity of responsibility.

Reply:
@masci @andrewkroh Could you put the list together?

Reply:
Derived from the elastic/integrations CODEOWNERS and git history.

```
/filebeat/module/apache @elastic/integrations
/filebeat/module/auditd @elastic/security-external-integrations
/filebeat/module/elasticsearch @elastic/integrations
/filebeat/module/haproxy @elastic/integrations
/filebeat/module/icinga @elastic/integrations
/filebeat/module/iis @elastic/integrations
/filebeat/module/kafka @elastic/integrations
/filebeat/module/kibana @elastic/integrations
/filebeat/module/logstash @elastic/integrations
/filebeat/module/mongodb @elastic/integrations
/filebeat/module/mysql @elastic/security-external-integrations
/filebeat/module/nats @elastic/integrations
/filebeat/module/nginx @elastic/integrations
/filebeat/module/osquery @elastic/security-asset-management
/filebeat/module/pensando @elastic/security-external-integrations
/filebeat/module/postgresql @elastic/integrations
/filebeat/module/redis @elastic/integrations
/filebeat/module/santa @elastic/security-external-integrations
/filebeat/module/system @elastic/ecosystem
/filebeat/module/traefik @elastic/integrations
/x-pack/filebeat/module/activemq @elastic/integrations
/x-pack/filebeat/module/aws @elastic/obs-cloud-monitoring
/x-pack/filebeat/module/awsfargate @elastic/obs-cloud-monitoring
/x-pack/filebeat/module/azure @elastic/obs-cloud-monitoring
/x-pack/filebeat/module/barracuda @elastic/security-external-integrations
/x-pack/filebeat/module/bluecoat @elastic/security-external-integrations
/x-pack/filebeat/module/cef @elastic/security-external-integrations
/x-pack/filebeat/module/checkpoint @elastic/security-external-integrations
/x-pack/filebeat/module/cisco @elastic/security-external-integrations
/x-pack/filebeat/module/coredns @elastic/security-external-integrations
/x-pack/filebeat/module/crowdstrike @elastic/security-external-integrations
/x-pack/filebeat/module/cyberarkpas @elastic/security-external-integrations
/x-pack/filebeat/module/cylance @elastic/security-external-integrations
/x-pack/filebeat/module/envoyproxy @elastic/security-external-integrations
/x-pack/filebeat/module/f5 @elastic/security-external-integrations
/x-pack/filebeat/module/fortinet @elastic/security-external-integrations
/x-pack/filebeat/module/gcp @elastic/security-external-integrations
/x-pack/filebeat/module/google_workspace @elastic/security-external-integrations
/x-pack/filebeat/module/ibmmq @elastic/integrations
/x-pack/filebeat/module/imperva @elastic/security-external-integrations
/x-pack/filebeat/module/infoblox @elastic/security-external-integrations
/x-pack/filebeat/module/iptables @elastic/security-external-integrations
/x-pack/filebeat/module/juniper @elastic/security-external-integrations
/x-pack/filebeat/module/microsoft @elastic/security-external-integrations
/x-pack/filebeat/module/misp @elastic/security-external-integrations
/x-pack/filebeat/module/mssql @elastic/integrations
/x-pack/filebeat/module/mysqlenterprise @elastic/security-external-integrations
/x-pack/filebeat/module/netflow @elastic/security-external-integrations
/x-pack/filebeat/module/netscout @elastic/security-external-integrations
/x-pack/filebeat/module/o365 @elastic/security-external-integrations
/x-pack/filebeat/module/okta @elastic/security-external-integrations
/x-pack/filebeat/module/oracle @elastic/security-external-integrations
/x-pack/filebeat/module/panw @elastic/security-external-integrations
/x-pack/filebeat/module/proofpoint @elastic/security-external-integrations
/x-pack/filebeat/module/rabbitmq @elastic/integrations
/x-pack/filebeat/module/radware @elastic/security-external-integrations
/x-pack/filebeat/module/snort @elastic/security-external-integrations
/x-pack/filebeat/module/snyk @elastic/security-external-integrations
/x-pack/filebeat/module/sonicwall @elastic/security-external-integrations
/x-pack/filebeat/module/sophos @elastic/security-external-integrations
/x-pack/filebeat/module/squid @elastic/security-external-integrations
/x-pack/filebeat/module/suricata @elastic/security-external-integrations
/x-pack/filebeat/module/threatintel @elastic/security-external-integrations
/x-pack/filebeat/module/tomcat @elastic/security-external-integrations
/x-pack/filebeat/module/zeek @elastic/security-external-integrations
/x-pack/filebeat/module/zookeeper @elastic/integrations
/x-pack/filebeat/module/zoom @elastic/security-external-integrations
/x-pack/filebeat/module/zscaler @elastic/security-external-integrations
```

Reply:
Thanks you @andrewkroh ❤️